clinica/iotools/utils/pipeline_handling.py

```diff
@@ -173,7 +173,7 @@ def pet_volume_pipeline(caps_dir, df, **kwargs):
                             atlas_df = pd.read_csv(atlas_path, sep='\t')
                             label_list = [group + '_' + atlas + '_ROI-' + str(x) for x in
                                           range(n_regions)]
-                            ses_df.loc[:, label_list] = atlas_df['mean_scalar'].as_matrix()
+                            ses_df.loc[:, label_list] = atlas_df['mean_scalar'].values
```

Comment:
According to [pandas API](https://pandas.pydata.org/pandas-docs/version/0.25.1/reference/api/pandas.DataFrame.values.html#pandas.DataFrame.values), they recommend to use `.to_numpy()` instead.